plugins/woocommerce/includes/widgets/class-wc-widget-cart.php

```diff
@@ -43,6 +43,60 @@ public function __construct() {
 		parent::__construct();
 	}
 
+	/**
+	 * Get list of Woo blocks that update the cart.
+	 *
```

Comment:
I'd explain the intention behind this curated list to leave out any confusion. 
One might ask himself why aren't the Cart and Checkout blocks included, as they can update the cart.